nin/backend/compress.js

```diff
@@ -92,7 +93,13 @@ function compress(projectPath, payload, callback) {
           'demo=bootstrap({layers:layers, camerapaths:camerapaths, onprogress: ONPROGRESS, oncomplete: ONCOMPLETE});' +
         '}' +
       '}' +
-    '}</script><canvas class=hide height='+ height + ' width=' + width + '></canvas><img src=# onload=z()><!--';
+    '}</script>' +
+    '<canvas class=hide height='+ height + ' width=' + width + '></canvas><img src=# onload=z()><!--';
+
+
+  var metadataChunks = Buffer.concat(metadata.map(function(data) {
+    return chunk('tEXt', new Buffer(data[0] + '\0' + data[1]));
```

Comment:
Turns out implementing iTXt support is rather difficult, and finding a tool that's able to show those segments even more so.
I suggest we wait until a need for it appears.